storage/cache/subtree_cache.go

```diff
@@ -129,8 +132,10 @@ func (s *SubtreeCache) splitNodeID(id storage.NodeID) ([]byte, Suffix) {
 	return a[:prefixSplit], sfx
 }
 
-// Preload populates the cache based on a specified set of NodeIDs.
-func (s *SubtreeCache) Preload(ids []storage.NodeID, getSubtrees func(id []storage.NodeID) ([]*storagepb.SubtreeProto, error)) error {
+// preload calculates the set of subtrees required to cover the passed in node
```

Comment:
By "cover" does that mean minimum set of subtrees that includes the nodes? Extra doc would be useful.